### DISCUSSION TITLE
How to change php.ini configs in wordpress application

### DISCUSSION THREAD

**Discussion by guiibraun** [coollabsio/coolify#1835] [Q&A] [answered]
Hello, I would like to know how I can update PHP settings such as memory limit and upload max size for Wordpress in cooliofy, normally I would access it via SFTP or something similar to do this but I don't know how to do it here

**Comment by dev-itzarun** [comment]
can any one please provide a solution to this

**Comment by mandrasch** [comment, marked as answer]
Hi @guiibraun @dev-itzarun,

are you deploying via Nixpacks? 

I found this [thread](https://help.railway.app/questions/max-upload-size-0bcea1f4), railway also uses Nixpacks. The thread mentions these two files

- https://gist.github.com/brody192/ed12b50fd39e373a28d75f25b100a87d
- https://gist.github.com/brody192/fcc1136b61c3de20609fa91ee384d47c

Not sure if [nginx.template.conf](https://gist.github.com/brody192/fcc1136b61c3de20609fa91ee384d47c#file-nginx-template-conf) is really needed.

Haven't tried yet if this works on coolify as well.  Other alternative would be to deploy via Dockerfile / Docker Compose.

**Comment by gmoz22** [comment]
Here's how to configure your Docker Compose to create an editable PHP `.ini` file in your Persistent Storages.


### 1. Configuration > Edit Compose File

Add the following entry:
```
services:
   wordpress:
      volumes: 
          - ./wordpress.ini:/usr/local/etc/php/conf.d/wordpress.ini
```

### 2. Save and Restart

### 3. Persistent Storages > Convert the `wordpress.ini` storage to a file

### 4. Edit the file with the PHP settings you want to override. e.g.:
```
file_uploads = On
memory_limit = 256M
upload_max_filesize = 64M
post_max_size = 64M
max_execution_time = 300
max_input_time = 1000
```

### 5. Save and Restart

:-)

**Reply by IRediTOTO** [reply]
love you <3

**Reply by SilkyChai** [reply]
Are you an angel? Did God send you?
